apps/super-app/src/services/DetailServices.ts

```diff
@@ -97,5 +98,6 @@ export function getTwc2Detail(): Twc2SupportDetail {
     ctaBlurb: 'Call on Mondays to Fridays (9am - 9pm), message any time',
     ctaButtonText: '+65 6297 7564',
     ctaLink: 'https://wa.me/6562977564',
+    whatsappLogoSrc: '/images/whatsapp-icon.svg',
```

Comment:
Hm, I think we should just reference the `src` string from the component since it's not like this property is going to come from Strapi.

We can clean this up later, but do leave a TODO comment if so.

Reply:
@glencbz what do you mean by this exactly? Understand that the image will be part of the FE asset instead, but what do you mean by `src` string from component?